src/main/java/karuna/karuna_backend/Controllers/ContentController.java

```diff
@@ -0,0 +1,35 @@
+package karuna.karuna_backend.Controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import karuna.karuna_backend.DTO.ContentDTO;
+import karuna.karuna_backend.Services.ContentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/contents")
+public class ContentController {
+
+    private final ContentService contentService;
+
+    @Operation(summary = "Get content by page", description = "Fetches content by page name, even if page doesn't " +
+            "exist it will always return default content")
+    @ApiResponse(
+            responseCode = "200",
+            description = "Content for page",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = ContentDTO.class)))
+    @GetMapping("/{page}")
+    public List<ContentDTO> getContentByPage(@PathVariable String page) {
```

Comment:
And now u returns list, why? We told u to use wrapper on collection.. So ContentDTO should contain the list and the other details of resposne body

Reply:
This can be package-private

Reply:
![Screenshot from 2024-05-02 08-25-50](https://github.com/karunaproject/be/assets/49255578/a81ab679-cc0d-4f5e-9c23-661b6a4f1500)

I'm already confused, so now I'll return the HashMap in contentDTO and if that's also wrong, I need a full description of what to return.
I will be unavailable for the next week, so feel free to make changes to this branch and merge it